src/app/home/components/home.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnDestroy } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+import { environment } from 'src/environments/environment';
+import { selectFavorites } from '../../store/selectors/pokemon.selectors';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.css']
+})
+export class HomeComponent implements OnDestroy {
+  private url = environment.pokemonImageUrl;
+  favoritesSubscription: Subscription;
+  slides = [];
+
+  constructor(private store: Store) {
+    this.favoritesSubscription = this.store.pipe(
+      select(selectFavorites))
+      .subscribe(
+        favoritePokemons => {
+          favoritePokemons.map(
+            pokemonId => {
+              this.slides.push({ path: this.url + pokemonId + '.png' });
```

Comment:
when you use map, you must use the returning value of the function map, don't use push